doc/upgrading.md

```diff
@@ -17,6 +18,24 @@ See the [NEWS](../NEWS.md) summary of changes for each version.
 * [Version 25](#version-25)
 * [Version 20](#version-20)
 
+## Version 32.0
+
+* [RMB-609](https://issues.folio.org/browse/RMB-609) Upgrade to Vert.x 4:
+  * Replace deprecated `io.vertx.core.logging.Logger` by
+    `org.apache.logging.log4j.Logger`, for example
+    `private static final Logger LOGGER = LogManager.getLogger(Foo.class)`
+    (using `getLogger()` without argument requires Multi-Release for
+    stack walking, see below).
```

Comment:
I didn't notice any further remarks on multi-release stack walking. What content is this referring to?

Reply:
@julianladisch ?

Reply:
Probably the Multi-Release section in 31.0.

Reply:
Yes, line 85.